test/bpf_tests/trf.proto

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
I'm curious about interactions using this trf format from the test prog to other systems. Is this just a nice way to get output formatted in a certain way, or can these be exported into some well-known system? How will we use it? This wasn't obvious to me from the commit message.

Reply:
I initially wrote my own serialization format, with the serializer in eBPF and a janky deserializer in Go. But I felt that creating a custom serialization format for a test suite was overkill and bloat. 

So I decided to use protobuf, which is compact on the wire due to the external `.proto` definitions which actually make it easier to craft messages in eBPF than something like JSON or MessagePack. And the Go code is free with ProtoBuf since all structs and deserialization is generated. Plus we already use protobuf in other places in the codebase so it doesn't add any dependencies. Seemed like a good choice all around.

Reply:
That seems like a good fit. How will we use this output? Do developers just read the protobuf output in a file somewhere?

Reply:
:100: nice.